theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-contribution.tsx

```diff
@@ -330,6 +343,12 @@ export class TraceViewerToolbarContribution implements TabBarToolbarContribution
             group: 'navigation',
             onDidChange: this.onMakerSetsChangedEvent
         });
+        registry.registerItem({
+            id: TraceViewerToolbarCommands.OPEN_TRACE_FILE.id,
+            command: TraceViewerToolbarCommands.OPEN_TRACE_FILE.id,
+            tooltip: TraceViewerToolbarCommands.OPEN_TRACE_FILE.label,
+            priority: 7
```

Comment:
The item registered with the command 'TraceViewerToolbarCommands.MARKER_SETS.id' also has the same id (7). Although these items won't cause conflict as they are not displayed in the same toolbar, you might consider adjusting the priorities so that its unique for each item

Reply:
Ah yes - i missed that! I will make them unique